src/webparts/peoplesearch/components/PeopleSearchBox/PeopleSearchBox.tsx

```diff
@@ -62,20 +62,28 @@ export class PeopleSearchBox extends React.Component<IPeopleSearchBoxProps,IPeop
           let query = queryText;
 
           this.setState({
-              searchInputValue: queryText,
               showClearButton: !isReset
           });
 
           let element = document.activeElement as HTMLElement;
           if (element) {
-              element.blur();
+              // element.blur();
           }
 
           // Notify the dynamic data controller
           this.props.onSearch(query);
       }
   }
 
+  public _onChange(value) {
+      this.setState({
+          searchInputValue: value
+      });
+      if (this.state.searchInputValue.length > 1) {
```

Comment:
if the goal is for this to work well with several hundred thousand users, >= 3 might be a better minimum length.